static/sass/styles.scss

```diff
@@ -18,3 +18,9 @@
   max-width: none !important;
   width: 100%;
 }
+
+// Fix for full-width global nav
+.global-nav .global-nav__strip {
+  margin: 0 auto !important;
+  max-width: $grid-max-width;
```

Comment:
This should be handled by the `maxWidth` option

Reply:
No, that's for the outer wrapper, the inner content needs to be constrained. Design & UX specified this. I'll submit a bug to the global-nav project to fix this upstream